README.md

```diff
@@ -42,6 +44,57 @@ use_frameworks!
 
 Then run `pod install` with CocoaPods 0.36 or newer.
 
+## Adding Runes as a Framework Dependency ##
+
+Runes can also be used as a low-overhead dependency for defining these
+operators for your own types. After linking to `Runes.swift`, you will be able
+to define implementations for your types, and your users will be able to
+import your framework to use these operators without needing to link against
+Runes directly.
```

Comment:
I still think this guy is a bit confusing. What about stating why you would want this first? Like "If you only want Runes' operator definitions and not their implementations for `Optional` and `Array`, then you can just link the `Runes.swift` file into you project instead of linking the entire framework. To do this you must use git submodules with Carthage, CocoaPods, or good ol' fashion git (not recommended) ... etc" something along those lines maybe?